App/Lettuce/Sources/Lettuce/MessageCenter/Message/GET/MessageRequest.swift

```diff
@@ -0,0 +1,24 @@
+// system
+import Foundation
+import HTTPTypes
+
+// local
+import Onion
+
+struct GetMessage: Request {
+    typealias Output = Message
+    
+    var path: String {
+        "https://api.allegro.pl/messaging/messages/\(messageId)"
```

Comment:
You just define the path part of the URL. The host part is defined in the API Client :)

Reply:
Fixed :)